go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -458,18 +453,14 @@ func (vs *vstreamer) parseEvent(ev mysql.BinlogEvent) ([]*binlogdatapb.VEvent, e
 				Type: binlogdatapb.VEventType_COMMIT,
 			})
 		case sqlparser.StmtDDL:
-			if mustSendDDL(q, params.DbName, vs.filter) {
+			if mustSendDDL(q, vs.cp.DBName(), vs.filter) {
 				vevents = append(vevents, &binlogdatapb.VEvent{
 					Type: binlogdatapb.VEventType_GTID,
 					Gtid: mysql.EncodePosition(vs.pos),
 				}, &binlogdatapb.VEvent{
 					Type:      binlogdatapb.VEventType_DDL,
 					Statement: q.SQL,
 				})
-				// Reload schema only if the DDL change is relevant.
-				// TODO(sougou): move this back to always load after
-				// the schema reload bug is fixed.
-				vs.se.ReloadAt(context.Background(), vs.pos)
```

Comment:
@sougou can you please verify that it is safe to remove this duplicated call? See https://github.com/vitessio/vitess/pull/7560/commits/6e40b88a376054e612179f81c51c69a36f1e96d3 for details

Reply:
@vmg, yes this can be removed. I thought I had already removed the duplication, thanks for catching this.

Reply:
Nice catch! The call should be removed.